apis/v1beta1/validation/common.go

```diff
@@ -50,15 +50,19 @@ func ValidateParentRefs(parentRefs []gatewayv1b1.ParentReference, path *field.Pa
 		}
 		if s, ok := parentRefsSectionMap[targetParentRefs]; ok {
 			if len(s[0]) == 0 || len(*targetSection) == 0 {
-				errs = append(errs, field.Required(path, "ParentRefs section names must all be set when ParentRefs includes 2 or more references to the same parent"))
+				errs = append(errs, field.Required(path.Child("parentRefs"), "sectionNames must be specified when more than one parentRef refers to the same parent"))
 				return errs
 			}
 			if utils.ContainsInSectionNameSlice(s, targetSection) {
-				errs = append(errs, field.Invalid(path.Index(i).Child("parentRefs").Child("sectionName"), targetSection, "must be unique when ParentRefs includes 2 or more references to the same parent"))
+				errs = append(errs, field.Invalid(path.Index(i).Child("parentRefs").Child("sectionName"), targetSection, "must be unique when parentRefs includes 2 or more references to the same parent"))
 				return errs
 			}
 		}
 		parentRefsSectionMap[targetParentRefs] = append(parentRefsSectionMap[targetParentRefs], *targetSection)
 	}
 	return errs
 }
+
+func ptrTo[T any](a T) *T {
```

Comment:
I'd usually agree with this but I avoided that for the following reasons:

1. Want to avoid creating a new package for a one line function.
2. Want to avoid supporting a utils package that might be used by implementations and have some appearance of backwards compatibility. Especially if the package is under `/apis`, it could appear like we're intending to provide the same compatibility guarantees as the rest of the `/apis` package.

Hopefully that makes sense.

Reply:
I see your argument @gyohuangxin, but I think in this instance, I agree with Rob.

Also, I'm pretty against `utils` packages in Go in general.

Reply:
@robscott It makes sense, thanks for your reply.